### ISSUE TITLE
fetching data from swiggy

### ISSUE THREAD

**Issue by sourabh321joshi** [akshaychavan23031998/Namaste_React_episode_5_-let-s-get-hooked-#1]
akshaychavan23031998
I'm having problem while fetching data from swiggy api can you help me

**Comment by akshaychavan23031998** [comment]
Use the mockdata file and watch the episode 4 and 5 u will able to do.

**Comment by akshaychavan23031998** [comment]
Ok ill help u

**Comment by sourabh321joshi** [comment]
Can you ping me on telegram my username @sourabh321joshii

**Comment by akshaychavan23031998** [comment]
Bhaii i am in episode 6 and i am also failed in API calling and data 😣😖,
how we can resolve this issue, upto episode 5 everything is correct 100%,
but on episode 6th i am tryed multiple times to get the data but failed, i
also don't no how to resolve this issue.

**Comment by sourabh321joshi** [comment]
I know how to resolve this issue

**Comment by akshaychavan23031998** [comment]
how broo ?? i tried my entire day but did not worked out for me, sweegy
valo ne restrict kiya haii other users koo our ek limit set kiya haii how
did u resolve that issue ??

**Comment by sourabh321joshi** [comment]
Same thing happened with me i also wasted my whole day while fetching live data but some how i managed to fetch I'm not able to fetch data of all restaurants only 7-8 restaurants data i fetched

**Comment by akshaychavan23031998** [comment]
Yes broo u r correct, there r only 7 to 8 restaurants i think, can u please
send me the url link if possible

**Comment by sourabh321joshi** [comment]
Which url link swiggy api ?

**Comment by akshaychavan23031998** [comment]
Yrl of swingy api link broo, from that u r getting 7 to 8 restaurant list

**Comment by sourabh321joshi** [comment]
"https://www.swiggy.com/dapi/restaurants/list/v5?lat=22.681711298285776&lng=75.83385474979877&is-seo-homepage-enabled=true&page_type=DESKTOP_WEB_LISTING"

**Comment by akshaychavan23031998** [comment]
broo its not working now, what happen is kii for 10 to 15 mins i where able
to see the object and extracted the data and immediately before 5 mins its
again not responding ?? i dont noo why its happening and coreproxy.io also
blocked mee dont no y 😫😖😩😢, i need a stable API for that i tried using
Rapid API fOR FREE It just gave me 3 API hits then it said i reached the
highest limit 😪😣😥, broo do u have any stable API so that it should work
for all ?

**Comment by sourabh321joshi** [comment]
Coreproxy.io blocked me also
I don't know any free or alternate api link
What error exactly you are getting?

**Comment by akshaychavan23031998** [comment]
Broo i am not getting any error message but i am not getting json data,
there i am getting error called not able to fetch the data, and showing
time out and something something.

**Comment by sourabh321joshi** [comment]
Push your code lemme see if i can help you

**Comment by akshaychavan23031998** [comment]
broo done, episode 6 is repo check that one, may be for around less than1
hr u will get the data but it will not fetch the data after that, its
happing same with me for last 2 days.

**Comment by sourabh321joshi** [comment]
type this path may be it will work
data.cards[4].card.card.gridElements.infoWithStyle.restaurants

**Comment by akshaychavan23031998** [comment]
I am still on 6th episode broo, if there API were stable then we haven't
face this much issue 🙃🙃.

**Comment by sourabh321joshi** [comment]
I'm getting basic error while fetching module i don't understand what to do I'm doing as sir taught still

**Comment by akshaychavan23031998** [comment]
Bro i feel that while extracting data u may be getting the error

**Comment by sourabh321joshi** [comment]
How you are managing are you continuing?

**Comment by akshaychavan23031998** [comment]
I wanna code while watching the videos but now only watching the videos and
understanding the concepts, what about you?

**Comment by sourabh321joshi** [comment]
There is no point on only watching videos in trying to fix error but it's been 2 days I'm stuck on the same thing

**Comment by akshaychavan23031998** [comment]
But what is the option we r left with ? The API is not stable, it will work
for 1 hr or 2 hrs not more than that, means we need to find out stable API
from Rapid API but the data is not proper it will only give u name of dish
and price and other 2 properties not more than that, so sticking at one
place we r going to not learn that much rather we will go and learn the
concepts at least and we can apply these concepts in other projects.

**Comment by sourabh321joshi** [comment]
Can you check episode 7 at 2 hr and try to install useParams hook and console what sir taught i tried everything to fix this but cannot try to console is it working or not

**Comment by akshaychavan23031998** [comment]
Ok ill let u know bro

**Comment by ankit-1012** [comment]
there are lot of issues with the api, while displaying the menu , the api has been structured differently
